webapi/src/test/scala/org/knora/webapi/slice/common/KnoraApiValueModelSpec.scala

```diff
@@ -58,17 +111,520 @@ object KnoraApiValueModelSpec extends ZIOSpecDefault {
+                                                       |       "@value": "4"
+                                                       |    }
+                                                       |  },
+                                                       |  "@context": {
+                                                       |    "ka": "http://api.knora.org/ontology/knora-api/v2#",
+                                                       |    "ex": "https://example.com/test#",
+                                                       |    "xsd": "http://www.w3.org/2001/XMLSchema#"
+                                                       |  }
+                                                       |}""".stripMargin)
+        content <- model.getValueContent()
+      } yield assertTrue(content == DecimalValueContentV2(ApiV2Complex, BigDecimal(4), None))
+    },
+    test("should parse BooleanValueContentV2") {
+      for {
+        model <- KnoraApiCreateValueModel.fromJsonLd("""{
+                                                       |  "@id": "http://rdfh.ch/0001/a-thing",
+                                                       |  "@type": "ex:Thing",
+                                                       |  "ex:someBool": {
+                                                       |    "@type": "ka:BooleanValue",
+                                                       |    "ka:booleanValueAsBoolean": {
+                                                       |       "@type": "xsd:boolean",
+                                                       |       "@value": "true"
+                                                       |    }
+                                                       |  },
+                                                       |  "@context": {
+                                                       |    "ka": "http://api.knora.org/ontology/knora-api/v2#",
+                                                       |    "ex": "https://example.com/test#",
+                                                       |    "xsd": "http://www.w3.org/2001/XMLSchema#"
+                                                       |  }
+                                                       |}""".stripMargin)
+        content <- model.getValueContent()
+      } yield assertTrue(content == BooleanValueContentV2(ApiV2Complex, true, None))
+    },
+    test("should parse GeomValueContentV2") {
+      for {
+        model <- KnoraApiCreateValueModel.fromJsonLd("""{
+                                                       |  "@id": "http://rdfh.ch/0001/a-thing",
+                                                       |  "@type": "ex:Thing",
+                                                       |  "ex:someGeom": {
+                                                       |    "@type": "ka:GeomValue",
+                                                       |    "ka:geometryValueAsGeometry": "{}"
```

Comment:
remark (non-blocking): the geometryValueAsGeometry `"{}"` seems suboptimal to me. I guess it's a valid value, but it's definitely not a good illustration of what these values should look like in reality, if we consider tests as "codified documentation".  
I don't want to clutter your test with something that is actually beside the point of the test. But I kind of like the idea of having meaningful values in these tests. How do you feel about this?